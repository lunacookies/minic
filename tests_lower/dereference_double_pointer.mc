func main {
	a := 5
	ap := &a
	app := &ap
	set ap = *app
}