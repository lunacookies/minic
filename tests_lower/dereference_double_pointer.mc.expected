func main
	var a i64
	var ap *i64
	var app **i64
	{
		set a = 5
		set ap = &a
		set app = &ap
		set ap = *app
	}
